City-of-Helsinki/drupal-repository
# Updated to php 8.2

## How to install
- `git checkout UHF-X-updates`
- `composer install`
- `docker compose up -d`

## How to test
- Run `cd /app` 
- Build the Satis database: `php console.php app:rebuild`
- Make sure the package index works: https://helfi-repository.docker.so/
